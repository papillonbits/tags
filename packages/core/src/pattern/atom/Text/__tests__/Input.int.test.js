import { regular } from './Input.int.story'

describe('<Input />', () => {
  describe('Render', () => {
    test('must match regular', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
