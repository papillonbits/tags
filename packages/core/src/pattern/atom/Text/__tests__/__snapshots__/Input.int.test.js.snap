// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> Render must match regular 1`] = `
<div>
  <input
    class="input"
    placeholder="Enter new text here"
    value=""
  />
</div>
`;
